team/_posts/2023-9-19-bibek-gyawali.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: member
+title: "Bibek Gyawali"
+category: member
+position: Rotation Student
+nickname: Bibek
+handle: Bibek
+email: 
+twitter: 
+github: 
+scholar: 
+image: /assets/images/team/bibek.jpg
+cv: 
+alum: false
+---
+A physics graduate seeking to bridge the worlds of Physics and Biology via the pursuit of truth and knowledge! I am deeply vested in learning the physics and biology behind life itself - in essence how the inanimate atoms and molecules give rise to vibrant, conscious behavioral patterns and ultimately life.
```

Comment:
This would also require editing. Can you work with Bibek to do that? He should remove any exclamation points(!), and rewrite the first sentence to be a full sentence (it is missing a subject and a verb). It also sounds very dramatic and grand, maybe tone it down a notch?

Reply:
Got it, I'll help Bibek make the edits.